README.md

```diff
@@ -200,3 +200,11 @@ Module Long Term Support (LTS)](http://github.com/CloudNativeJS/ModuleLTS) polic
 | 4.x     | End-of-Life     | Dec 2016  | Apr 2019 |
 
 Learn more about our LTS plan in [docs](https://loopback.io/doc/en/contrib/Long-term-support.html).
+
+## swagger-ui vulnerabilities
```

Comment:
Can we make this section easier to discover please, for example by moving it to the top of README, or adding a short paragraph at the top with a link to this section?

Reply:
I moved it right under `Basic Usage` now.